### PR TITLE
Assigment4CarlosD21

### PR DESCRIPTION
**Pull request by CarlosD21**

### CODE REVIEW

File: Assignment4/CarlosD21-190382/Task06.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1hSv61ejXESrp34CUz1PqZCsoA43tzc8y
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib
```
Comment (dgarijo) [inline Assignment4/CarlosD21-190382/Task06.py]: won't work in python. This is notebook magic

File: Assignment4/CarlosD21-190382/Task06.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+g.add((ns.JaneSmith, vcard.FN, Literal("Jane Smith")))
+g.add((ns.Jane, vcard.Given, Literal("Jane")))
+g.add((ns.Smith, vcard.Family, Literal("Smith")))
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.5: Add UPM as the university where John Smith works**"""
+
+g.add((ns.UPM, RDF.type, ns.University))
+g.add((ns.JohnSmith, vcard.work, ns.UPM))
```
Comment (dgarijo) [inline Assignment4/CarlosD21-190382/Task06.py]: vcard.work does not exist. Create a property under the ns namespace

File: Assignment4/CarlosD21-190382/Task07.py [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    print(s2)
+
+print("SPARQL:")
+q2= prepareQuery('''SELECT ?sub WHERE { 
+    { 
+      ?sub rdf:type ns:Person. 
+    } 
+    UNION {
+      ?subclass rdfs:subClassOf ns:Person.
+      ?sub rdf:type ?subclass
```
Comment (dgarijo) [inline Assignment4/CarlosD21-190382/Task07.py]: Missing all the subclass hierarchy (use chained properties)

File: Assignment4/CarlosD21-190382/Task07.py [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+      ?sub ?obj ?pred
+    } UNION {
+      ?sub rdf:type ?x.
+      ?x rdfs:subClassOf ns:Person.
+      ?sub ?obj ?pred.
+    }
+  }
+  ''',
+  initNs = { "rdfs": RDFS,"rdf": RDF, "ns": ns}
+)
```
Comment (dgarijo) [inline Assignment4/CarlosD21-190382/Task07.py]: Same as above

**Review by dgarijo — CHANGES_REQUESTED** [review]
Small changes requested

File: Assignment4/CarlosD21-190382/Task07.py (hunk 3 above)
Comment (CarlosD21) [reply Assignment4/CarlosD21-190382/Task07.py]: I have done some changes but I don´t understand very well this point.

**Comment by dgarijo** [comment]
You have deleted your tasks and added notebooks instead, which are not accepted. Open a new PR, as this is not ok.

**Pull request closed without merging**